appdata/currently_reading.csv

```diff
@@ -2,25 +2,27 @@
 "1","Harry Potter and The Sorcerer's Stone","J.K. Rowling","tester1","0","-1","4","anotherUser","false","false"
```

Comment:
All appdata files (csv files) probably do not require checking/have just been updated due to my debugging. When things went wonky in tests I went back and altered the csv data so that there is nothing contradictory (like a currently_reading.csv entry having "true" for has_open_search when the SearchPost entry was deleted from search_posts.csv) and hopefully I didn't miss anything.